From 6c7fe42cb8e67f369d2cba3872a9b45d8380ae07 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Thu, 9 Mar 2023 13:27:06 +0100
Subject: [PATCH] Exclude "test" package from being installed

Add `test` to excludes for `find_packages()`, to prevent the package
from wrongly installing the `test` directory into site-packages, e.g.:

    /usr/lib/python3.11/site-packages/test
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index a124748..36db5d7 100644
--- a/setup.py
+++ b/setup.py
@@ -53,7 +53,7 @@ setup (
 
   url='https://github.com/Quansight-Labs/accessible-pygments',
 
-  packages=find_packages(),
+  packages=find_packages(exclude=['test']),
   install_requires=[
     'pygments >= 1.5'
   ],
-- 
2.39.2

